### ISSUE TITLE
[LibreSSL] Build failure for lang/rust

### ISSUE THREAD

**Issue by lattera** [HardenedBSD/hardenedbsd-ports#99]
Build log attached.

[rust-1.19.0_2.log.txt](https://github.com/HardenedBSD/hardenedbsd-ports/files/1309526/rust-1.19.0_2.log.txt)

**Comment by raymontag** [comment]
Hello guys,

I tried to look into this. From l.5570 ongoing we can see that the reason for the failure was rust-openssl which panicked while building because of an unsupported LibreSSL version:

`This crate is only compatible with OpenSSL 1.0.1, 1.0.2, and 1.1.0, or LibreSSL
2.5.0, 2.5.1, 2.5.2, 2.5.3, and 2.5.4, but a different version of OpenSSL was
found. The build is now aborting due to this version mismatch.`

But in September we already had LibreSSL 2.6.X. 

However, the newest version of rust-openssl supports the currently used LibreSSL-version. So I tried to build lang/rust-1.22.1 on 11.1-STABLE and it worked :)

So maybe you just have to retrigger the build of the current lang/rust version and it will build again. Personally, I would really appreciate it because I don't want to build firefox from ports :)

See my log for reference:

[rust-1.22.1.log.txt](https://github.com/HardenedBSD/hardenedbsd-ports/files/1516823/rust-1.22.1.log.txt)
